src/stateManagement/textStyle.js

```diff
@@ -1,38 +1,56 @@
-let defaultFontSize = 15,
-  defaultFont = `${defaultFontSize}px Arial`,
-  defaultBackgroundColor = 'transparent';
+const textStyle = {
+  fontSize: 15,
+  fontFamily: 'Arial',
+  backgroundColor: 'transparent',
+};
 
+// Deprecate
 function setFont(font) {
-  defaultFont = font;
+  const split = font.split('px ');
+
+  if (split.length === 2) {
```

Comment:
Is it going to be deprecate?
Given param is restritected to: fontSize|fontFamily:
1. Would be nice if we parametrize the position each item
2. Would be nice if there is a minimum validation
3. Can we default values to initial textStyle values?
3.1 Is it possible to accept a list of font-family names and/or generic-family names?

Reply:
Yes, it is going to be deprecated.
Updated the code and added a more detailed jsDoc with explanations.

Reply:
3. As we discussed, it will not worth changing the original function as it's being deprecated.